### ISSUE TITLE
Confising documentation for run --read-only-tmpfs=true

### ISSUE THREAD

**Issue by dilyanpalauzov** [containers/podman#17103]
### Issue Description

man podman-run prints:
```
--read-only-tmpfs
  If container is running in --read-only mode, then mount a read-write tmpfs on /run, /tmp,
  and /var/tmp. The default is true.
```

This implies, that `podman run --read-only` and `podman run --read-only --read-only-tmpfs=true` are the same thing.  But `--read-only-tmpfs=true` means, that /tmp is read-only, as this is what the wording suggests, without looking at the documentation.  So `read-only-tmfps=true` means the /tmp is read-write, and `read-only-tmpfs=false` means /tmp is true.

It  is very confusing to have read-only=true meaning read-write.

Please change the option names to avoid this confusion.

### Steps to reproduce the issue

N/A

### Describe the results you received

N/A

### Describe the results you expected

N/A

### podman info output

```shell
Client:       Podman Engine
Version:      4.4.0-dev
API Version:  4.4.0-dev
Go Version:   go1.19.4
Git Commit:   93118464d4692a3682701cdad394373652d25035
Built:        Fri Jan 13 09:09:44 2023
OS/Arch:      linux/amd64
```

### Podman in a container

No

### Privileged Or Rootless

Rootless

### Upstream Latest Release

Yes

### Additional environment details

Additional environment details

### Additional information

Additional information like issue happens only occasionally or issue happens with a particular architecture or on a particular setting

**Comment by rhatdan** [comment]
Yes the wording sucks, but what it is supposed to show is
podman run --read-only --read-only-tmpfs=false 
Means that everything is read-only, it might be clearer if we rename the option to --read-write-tmpfs.  We need a better option that is not confusing.

**Comment by vrothberg** [comment]
@rhatdan I'll assign this to you.  I am sure we can find aliases on the CLI that are less confusing.

**Comment by rhatdan** [comment]
Yes I am looking for suggestions.  --read-write-tmpfs? Any others?

**Comment by vrothberg** [comment]
` --read-write-tmpfs` SGTM but now I feel torn (apologies in advance).  What would a `-- --read-write-tmpfs=false` mean?  Couldn't it be equally confusing?

Maybe we can find a way to improve the docs instead?

**Comment by rhatdan** [comment]
--rw-tmpfs-whenreadonly

**Comment by hauges** [comment]
What is the current state of this issue?

On my system, I am running podman version 4.6.1. When I set `--read-write-tmpfs=false`, it does as the flag suggests (sets /tmp, etc. as rw). When I set `--read-write-tmpfs=true` it does the opposite. This, however, is the opposite of what the documentation states.

I am further confused on the state of this issue given the discussion on these other issues and PRs:
https://github.com/containers/podman/pull/20235
https://github.com/containers/podman/issues/20225

**Comment by rhatdan** [comment]
`--read-only --read-only-tmpfs=false` Should make everything other then volumes read-only.
`--read-only --read-only-tmpfs=true` Should allow users to write to tmfs mounted on /tmp, /dev/shm ...

$ podman run --read-only --read-only-tmpfs=true alpine touch /tmp/foobar
$ podman run --read-only --read-only-tmpfs=false alpine touch /tmp/foobar
touch: /tmp/foobar: Read-only file system

**Comment by rhatdan** [comment]
Fixed in podman 5.0

**Comment by hauges** [comment]
> Fixed in podman 5.0

Are you saying the behavior has been fixed according to the use case you provided above in 5.0? If so, is there different behavior in 4.6.1?
